### PR TITLE
Fixes and caveats

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: docs/spicedb/operator.md [hunk 1]
```diff
@@ -121,16 +121,17 @@ There may be exceptions to this rule, but they will be documented in release not
...
+| dispatchUpstreamCASecretName | The name of a Kubernetes secret in the same namespace to use as the TLS CA validation. This should be the CA cert that was used to issue the cert in `tlsSecretName` | string                      |
+| datastoreTLSSecretName       | The name of a Kubernetes secret containing a TLS secret to use when connecting to the datastore.                                                                     | string                      |
+| spannerCredentials           | The name of a Kubernetes secret containing credentials for talking to Cloud Spanner. Typically, this would not be used, in favor of workload identity.               | string                      |
+| extraPodLabels               | A set of additional labels to add to the spicedb pods.                                                                                                               | string or map[string]string |
+| extraPodAnnotations          | A set of additional annotations to add to the spicedb pods.                                                                                                          | string or map[string]string |
```
Comment (vroldanbet) [inline docs/spicedb/operator.md:134]: this reminds me we haven't made the new release including this feature 👀 
https://github.com/authzed/spicedb-operator/releases

**Review by vroldanbet — APPROVED** [review]
Looks great! just a few nits

File: docs/reference/caveats.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+In the above example, a relationship can be written for the `viewer` relation to a `user` without a caveat OR with the `ip_allowlist` caveat.
+
+To make the caveat **required**, the `user |` can be removed.
+
+## Writing relationships with caveats and context
+
+When writing a relationship for a relation, both the caveat and a portion of the “context” can be specified:
+
+```
+WriteRelationshipsRequest {
```
[the remark this answers is not in the document]
Comment (josephschorr) [reply docs/reference/caveats.md:177]: Probably eventually, but for now I just wanted to get the basics in

File: docs/reference/caveats.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+        }
+      }
+    }
+  ]
+}
+```
+
+A few important notes:
+
+- The **Context** of a caveat here is optional: if empty, then only the context specified on a CheckPermission request will be used.
```
[the remark this answers is not in the document]
Comment (josephschorr) [reply docs/reference/caveats.md:199]: https://github.com/authzed/docs/issues/79

File: docs/spicedb/configuring-dispatch.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+[consistent hash load balancing for grpc]: https://authzed.com/blog/consistent-hash-load-balancing-grpc/
+
+## Configuring Dispatch in Kubernetes
+
+If not using the [SpiceDB Operator], the `dispatch-upstream-addr` should be of the form `kubernetes:///spicedb.default:50053` where `spicedb.default` is the Kubernetes `Service` in which SpiceDB is accessible.
+
+## Configuring Dispatch in non-Kubernetes Environments
+
+:::warning
+Non-Kubernetes based dispatching relies upon DNS updates, which means it can become stale if DNS is changing. This is not recommended unless DNS updates are rare.
```
[the remark this answers is not in the document]
Comment (josephschorr) [reply docs/spicedb/configuring-dispatch.md:26]: https://github.com/authzed/docs/issues/79

File: docs/spicedb/operator.md (hunk 1 above)
Comment (josephschorr) [reply docs/spicedb/operator.md:134]: D'oh. Yeah, we need to do that.

**Review by vroldanbet — APPROVED** [review]

**Review by vroldanbet — APPROVED** [review]

**Pull request merged**